Optimize command examples in Lisk core v4.0.0 (#895)
LiskHQ/lisk-core: docs/antora/modules/ROOT/pages/setup/source.adoc

This section details how to install Lisk Core from the source code.
When completed, a functioning node on the Lisk network will exist.

Executing the following commands below will enable you to clone the repository, navigate into the `lisk-core` root folder, check out the latest release tag, and install the dependencies.

.Clone the repository
[source,bash]
----
git clone https://github.com/LiskHQ/lisk-core.git
----

> .Navigate into the `lisk-core` root folder

Reviewer: ```suggestion
.Navigate into the "lisk-core" root folder
```

[source,bash]
----

> cd lisk-core/docker

Reviewer 2: ```suggestion
cd lisk-core
```

----

.Check out the latest release tag
[source,bash]
----

> git checkout v4.0.0 -b v4.0.0

Reviewer 2: ```suggestion
git checkout v4.0.0
```

----
NOTE: Please check for the latest release in {url_core_releases}[Lisk Core releases^].

.Install dependencies
[source,bash]
----
yarn install --frozen-lockfile
----
.Build the binary
[source,bash]
----
yarn build
----

> To set up the Lisk core alias, add the following line to your `~/.profile` and run `source ~/.profile`.

Reviewer 2: ```suggestion
Set up the `lisk-core` alias: Append the following line to your `~/.profile` file and run `source ~/.profile`.
```

> Set up the `lisk-core` alias: Append the following line to your `~/.profile` file and run `source ~/.profile`.
> [source,bash]
> ----
> alias lisk-core=/home/lisk/lisk-core/bin/run
> ----

Reviewer 2: ```suggestion
Set up the `lisk-core` alias: Append the following line to your `~/.profile` file and run `source ~/.profile`.
[source,bash]
----
alias lisk-core=/home/lisk/lisk-core/bin/run
----
NOTE: In the command above, please replace `/home/lisk/lisk-core` with the absolute path to your "lisk-core" directory. To find the exact path on your system, change to your lisk-core directory and run the `pwd` command.
```
